lib/modules/hideChildComments.js

```diff
@@ -6,12 +6,17 @@ export const module = {};
 module.moduleID = 'hideChildComments';
 module.moduleName = 'Hide All Child Comments';
 module.category = 'Comments';
-module.description = 'Allows you to hide all comments except for replies to the OP for easier reading.';
+module.description = 'Allows you to hide child comments.';
 module.options = {
 	automatic: {
 		type: 'boolean',
 		value: false,
-		description: 'Automatically hide all but parent comments, or provide a link to hide them all?',
+		description: 'Hide children for all comments by default.',
+	},
+	noNested: {
```

Comment:
I think the inverse of this would be easier to understand (no double-negative when it's disabled), i.e. "nested: Hide children for nested comments when hiding all child comments."